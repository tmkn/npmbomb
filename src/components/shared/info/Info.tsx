/** @jsx jsx */
import { jsx, css } from "@emotion/core";

import { sansSerifFont, primaryColorDark, secondaryColorLight, mq } from "../../../css";

const infoStyle = css({
    [mq[0]]: {
        display: "flex",
        flexDirection: "column",
        fontFamily: `"${sansSerifFont}"`,
        padding: "2rem",
        backgroundColor: `${secondaryColorLight}`,
        color: `${primaryColorDark}`,
        margin: 0,
        marginBottom: "2rem"
    }
});

export const Info: React.FC = ({ children }) => {
    return <p css={infoStyle}>{children}</p>;
};
