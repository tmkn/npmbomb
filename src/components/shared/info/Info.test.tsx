import React from "react";
import { render, fireEvent, waitForElement } from "@testing-library/react";

import { Info } from "./Info";

describe("<Info />", () => {
    test("matches snapshot", () => {
        const { asFragment } = render(<Info>Hello</Info>);

        expect(asFragment()).toMatchSnapshot();
    });
});
