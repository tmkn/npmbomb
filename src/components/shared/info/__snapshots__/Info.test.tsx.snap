// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Info /> matches snapshot 1`] = `
<DocumentFragment>
  <p
    class="css-1axajee"
  >
    Hello
  </p>
</DocumentFragment>
`;
